src/OSVRTrackedDevice.cpp

```diff
@@ -436,6 +436,16 @@ vr::DistortionCoordinates_t OSVRTrackedDevice::ComputeDistortion(vr::EVREye eye,
     coords.rfBlue[0] = coords_blue[0];
     coords.rfBlue[1] = 1.0f - coords_blue[1];
 
+	// offset for center of projection
+	auto x_offset = 0.5 - distortion_parameters.m_distortionCOP[0];
+	auto y_offset = 0.5 - distortion_parameters.m_distortionCOP[1];
+	coords.rfRed[0] = coords.rfRed[0] + x_offset;
```

Comment:
This allows the (_u_, _v_) coordinates to exceed the [0.0, 1.0] range, doesn't it?